bot/commands/main.go

```diff
@@ -128,12 +128,14 @@ func replyErr(e *gateway.InteractionCreateEvent, context string, err error) erro
 	return ephemeralReply(e, "Something went wrong: ```\n"+err.Error()+"```")
 }
 
-func replyWithFlags(e *gateway.InteractionCreateEvent, flags api.InteractionResponseFlags, content string) error {
+func replyWithFlags(e *gateway.InteractionCreateEvent, flags api.InteractionResponseFlags, content string, embeds *[]discord.Embed) error {
 	return s.RespondInteraction(e.ID, e.Token, api.InteractionResponse{
 		Type: api.MessageInteractionWithSource,
 		Data: &api.InteractionResponseData{
-			Content: option.NewNullableString(content),
-			Flags:   flags,
+			Content:         option.NewNullableString(content),
+			Flags:           flags,
+			Embeds:          embeds,
+			AllowedMentions: &api.AllowedMentions{},
```

Comment:
This PR also disallows all mentions, I don't think we have any feature that should ever mention anyone, do we?

Reply:
ping manti command